test_env.py

```diff
@@ -0,0 +1,18 @@
+import chess
+from chess_env import ChessEnv
+
+env = ChessEnv(r"C:\Users\CSC\Desktop\Leiden\stockfish-10-win\stockfish-10-win\Windows\stockfish_10_x64.exe", player_color =chess.WHITE)
```

Comment:
Apparently the path string must be a regex string for some reason. Mimicking the same format escaped that permission error I seemed to be facing earlier.